### PR TITLE
feat: add new rpc method to get wallet information and show warning for the user in post install page

### PR DESCRIPTION
**Pull request by pedroferreira1**
### Motivation

It's important for dApps to know the connected network and first address of the wallet. We decided to create this rpc method that doesn't need approval to share this information and show a warning of it to the user after the snap is installed.

### Acceptance Criteria

- Create rpc method to get wallet information (network and address0 for now)
- Create post install page with docs information and warning about dapps being able to get network and address0.

image

### Checklist
- [x] If you are requesting a merge into `master`, confirm this code is production-ready and can be included in future releases as soon as it gets merged
- [x] Make sure either the unit tests and/or the QA tests are capable of testing the new features
- [x] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: packages/hathor-rpc-handler/__tests__/rpcMethods/getWalletInformation.test.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      address0: 'test-address',
+    });
+    expect(mockWallet.getNetwork).toHaveBeenCalled();
+    expect(mockWallet.getAddressAtIndex).toHaveBeenCalledWith(0);
+  });
+
+  it('should work with different network', async () => {
+    const mockMainnetWallet = {
+      getNetwork: jest.fn().mockReturnValue('mainnet'),
+      getAddressAtIndex: jest.fn().mockResolvedValue('HTestAddress123'),
```
Comment (tuliomir) [inline packages/hathor-rpc-handler/__tests__/rpcMethods/getWalletInformation.test.ts:69]: nitpick(non-blocking): For a more future-proof test and easier auditing, we could throw if this function is called with anything other than zero.
```suggestion
      getAddressAtIndex: jest.fn().mock(addressIndex => {
        if (addressIndex === 0) return 'HTestAddress123';
        throw new Error('Forbidden');
      }),
```

**Review by andreabadesso — APPROVED** [review]

File: packages/snap/src/dialogs/install.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      content: (
+        <Container backgroundColor='alternative'>
+          <Box>
+            <Heading>Installation successful</Heading>
+            <Text>
+              You can access the full documentation on the <Link href="https://docs.hathor.network/explanations/features/metamask-snap">Hathor docs</Link>
+            </Text>
+            <Text>
+              The dApp you are connecting to will have access to your first address and the network you are connected to
+            </Text>
```
[the remark this answers is not in the document]
Comment (pedroferreira1) [reply packages/snap/src/dialogs/install.jsx:25]: We don't have a component for list in the snap sdk.

**Review by tuliomir — APPROVED** [review]
